zerver/webhooks/taiga/view.py

```diff
@@ -48,19 +37,19 @@ def api_taiga_webhook(request: HttpRequest, user_profile: UserProfile,
 
 templates = {
     'epic': {
-        'create': u':package: %(user)s created epic **%(subject)s**',
-        'set_assigned_to': u':busts_in_silhouette: %(user)s assigned epic **%(subject)s** to %(new)s.',
-        'unset_assigned_to': u':busts_in_silhouette: %(user)s unassigned epic **%(subject)s**',
-        'changed_assigned_to': u':busts_in_silhouette: %(user)s reassigned epic **%(subject)s**'
+        'create': u' %(user)s created epic **%(subject)s**',
+        'set_assigned_to': u' %(user)s assigned epic **%(subject)s** to %(new)s.',
+        'unset_assigned_to': u' %(user)s unassigned epic **%(subject)s**',
+        'changed_assigned_to': u' %(user)s reassigned epic **%(subject)s**'
         ' from %(old)s to %(new)s.',
-        'blocked': u':lock: %(user)s blocked epic **%(subject)s**',
-        'unblocked': u':unlock: %(user)s unblocked epic **%(subject)s**',
-        'changed_status': u':chart_increasing: %(user)s changed status of epic **%(subject)s**'
+        'blocked': u' %(user)s blocked epic **%(subject)s**',
+        'unblocked': u' %(user)s unblocked epic **%(subject)s**',
+        'changed_status': u' %(user)s changed status of epic **%(subject)s**'
         ' from %(old)s to %(new)s.',
-        'renamed': u':notebook: %(user)s renamed epic from **%(old)s** to **%(new)s**',
-        'description_diff': u':notebook: %(user)s updated description of epic **%(subject)s**',
-        'commented': u':thought_balloon: %(user)s commented on epic **%(subject)s**',
-        'delete': u':cross_mark: %(user)s deleted epic **%(subject)s**',
+        'renamed': u' %(user)s renamed epic from **%(old)s** to **%(new)s**',
+        'description_diff': u' %(user)s updated description of epic **%(subject)s**',
+        'commented': u' %(user)s commented on epic **%(subject)s**',
+        'delete': u' %(user)s deleted epic **%(subject)s**',
     },
     'relateduserstory': {
         'create': (u':package: %(user)s added a related user story '
```

Comment:
You forgot to remove the `:package:`  emoji here, which is why your tests are failing! :)